provisioners/roles/ckan/tasks/main.yml

```diff
@@ -82,6 +99,14 @@
   sudo: yes
   notify: restart apache
 
+
+- name: create crontab for sending activity stream emails
+  cron: name="ckan_send_emails"
+        minute={{ckan_email_flush_interval}}
+        job="echo $(date) >> {{ckan_log_dir}}/email_flush.log && echo '{}' | {{ckan_venv}}/bin/paster post -c {{ckan_config_dir}}/config.ini /api/action/send_email_notifications >> {{ckan_log_dir}}/email_flush.log && echo '' >> {{ckan_log_dir}}/email_flush.log"
```

Comment:
Only thought here is wondering if it's worth pulling this command into it's own script so we can include it as a template.  On one hand, it would make maintenance easier and more clear.  But honestly it seems like such a throwaway line that it's not really even worth it.  

Yeah, don't bother, not worth it.